superchain/configs/mainnet/op.yaml

```diff
@@ -1,4 +1,5 @@
-name: OP-Mainnet
+name: OP Mainnet
+short_name: oeth
```

Comment:
Should we use `opeth` here instead? I feel like `op` is a commonly used prefix for optimism across docs/repos

Reply:
This is the short name already installed in the upstream ethereum-lists/chains repo. I agree that `op` is the better prefix -- we aren't using the short name downstream ourselves for anything at the moment. However, it is part of the upstream check so we need to store it.